spec/presenters/announcements_presenter_spec.rb

```diff
@@ -1,16 +1,12 @@
-require "test_helper"
-
-describe AnnouncementsPresenter do
+RSpec.describe AnnouncementsPresenter do
   describe "slug with a locale" do
-    before :each do
-      @presenter = AnnouncementsPresenter.new("boris-johnson.cy", filter_key: "people")
-    end
+    let(:presenter) { described_class.new("boris-johnson.cy", filter_key: "people") }
 
     it "uses a locale-less slug in the links" do
-      links = @presenter.links
-      assert_equal links[:email_signup], "/email-signup?link=/government/people/boris-johnson"
-      assert_equal links[:subscribe_to_feed], "/search/news-and-communications.atom?people=boris-johnson"
-      assert_equal links[:link_to_news_and_communications], "/search/news-and-communications?people=boris-johnson"
+      links = presenter.links
+      expect("/email-signup?link=/government/people/boris-johnson").to eq(links.[](:email_signup))
```

Comment:
`.[]` in a few places 😬

Reply:
I found a couple more in an old model spec, so have added those to this commit too